index.html

```diff
@@ -388,142 +388,145 @@
-                <li><code>pointerup</code></li>
-                <li><code>pointermove</code></li>
-                <li><code>pointerover</code></li>
-                <li><code>pointerout</code></li>
-            </ul>
-            
-            <p>The target object at which the event is fired is determined as follows:
-            <ul>
-                <li>If the <a>pointer capture target override</a> has been set for the pointer, 
-                    <ul>
-                        <li>Set the <code>relatedTarget</code> attribute of the event to <code>null</code>.</li>
-                        <li>Fire the event to the <a>pointer capture target override</a> object.</li>
-                    </ul>
-                </li>
-                <li>Otherwise, fire the event to the object returned by normal hit test mechanisms (out of scope for this specification).</li>
-            </ul>
-            <h4>Process Pending Pointer Capture</h4>
-            <p>Whenever a user agent is to fire a Pointer Event that is not <code>gotpointercapture</code> or <code>lostpointercapture</code>, it must first run these steps:</p>
-            <ol>
-                <li>If the <a>pointer capture target override</a> for this pointer is set and is not equal to the <a>pending pointer capture target override</a>, then fire a pointer event named <code>lostpointercapture</code> at the <a>pointer capture target override</a> node.
-                    <ul>
-                        <li>Further, if the <a>pending pointer capture target override</a> is not set and, the <a>pointer capture target override</a> is not equal to the hit test node for the pointer event which invoked this process, then fire a pointer event named <code>pointerover</code> and a pointer event named <code>pointerenter</code> at the hit test node.</li>
-                    </ul>
-                </li>
-                <li>If the <a>pending pointer capture target override</a> for this pointer is set and is not equal to the <a>pointer capture target override</a>, then fire a pointer event named <code>gotpointercapture</code> at the <a>pending pointer capture target override</a>. 
-                    <ul>
-                        <li>Further, if the <a>pointer capture target override</a> is not set and, the <a>pending pointer capture target override</a> is not equal to the hit test node for the pointer event which invoked this process, and the hit test node has received <code>pointerover</code> and <code>pointerenter</code> events, then fire a pointer event named <code>pointerout</code> and a pointer event named <code>pointerleave</code> at the hit test node.</li>
-                    </ul>
-                </li>
-                <li>Set the <dfn>pointer capture target override</dfn> to the <a>pending pointer capture target override</a>, if set. Otherwise, clear the <a>pointer capture target override</a>.</li>
-            </ol>
+            <h2>List of Pointer Events</h2>
```

Comment:
Nit: I prefer the previous title "Pointer Event Types" (I think the "list" was referring to the summary in the table, doesn't seem to describe the entire section to me).